Dockerfile-debian.template

```diff
@@ -31,46 +31,67 @@ RUN set -ex; \
 	apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false; \
 	rm -rf /var/lib/apt/lists/*
 
+# set recommended PHP.ini settings
+# see https://secure.php.net/manual/en/opcache.installation.php
+RUN { \
+		echo 'opcache.memory_consumption=128'; \
+		echo 'opcache.interned_strings_buffer=8'; \
+		echo 'opcache.max_accelerated_files=4000'; \
+		echo 'opcache.revalidate_freq=2'; \
+		echo 'opcache.fast_shutdown=1'; \
+		echo 'opcache.enable_cli=1'; \
+	} > /usr/local/etc/php/conf.d/opcache-recommended.ini; \
+	\
+	{ \
+		echo 'session.cookie_httponly = 1'; \
+		echo 'session.use_strict_mode = 1'; \
+	} > /usr/local/etc/php/conf.d/session-strict.ini; \
+	\
+	{ \
+		echo 'allow_url_fopen = Off'; \
+		echo 'max_execution_time = 600'; \
+		echo 'memory_limit = 512M'; \
+	} > /usr/local/etc/php/conf.d/phpmyadmin-misc.ini
+
 # Calculate download URL
 ENV VERSION 4.8.5
 ENV URL https://files.phpmyadmin.net/phpMyAdmin/${VERSION}/phpMyAdmin-${VERSION}-all-languages.tar.xz
 LABEL version=$VERSION
 
 # Download tarball, verify it using gpg and extract
 RUN set -ex; \
-    fetchDeps=" \
```

Comment:
I believe your editor has converted many of these spaces to tabs, is that intentional? Throughout phpMyAdmin's repositories, we try to stick with spaces unless there's a compelling reason to use a tab isntead.

Reply:
I converted them to spaces now.